growth/too/views.py

```diff
@@ -783,6 +783,18 @@ def plan_manual():
             telescope = form.telescope.data
             json_data, queue_name = get_json_data_manual(form)
 
+            mess = 'Target list empty. None of the requested fields ' + \
+                   'have references in the band(s) requested.'
+
+            if not json_data["targets"] and bool(form.references.data):
```

Comment:
There is no need for this function to have multiple exit points, or even multiple nested `if` statements, if you restructure the logic like this:

```python
if not(request.method == 'POST' and form.validate()):
    pass
elif ...:
    flash(...)
elif ...:
    flash(...)
else:
    group(...).delay()
    flash(...)
return render_template(...)
```

Reply:
check again?